circle.yml

```diff
@@ -0,0 +1,20 @@
+machine:
+  python:
+    version: 2.6.8
```

Comment:
Unfortunately testing multiple python versions in CircleCI is a bit more involved than Travis, so I went with the lowest version we support for now.